.gitignore

```diff
@@ -1,18 +1,12 @@
-*~
-.#*
-*#
-model.log
-*/input.csv
-__pycache__
-.python-version
-*sqlite3
+*sqlite3*
```

Comment:
This looks a little bit too broad.  (I've got a file a project called `sqlite_wrapper.py`, which would almost get caught by this.)  Could we put the database files in a directory, and add that directory to .gitignore?

Reply:
Good point 👍